topics/contributing_to_the_framework.adoc

```diff
@@ -51,11 +51,24 @@ We encourage contributions to the _Community Collaboration Guide_. In particular
 
 * You must have downloaded, installed, and configured Git on your desktop.
 
-* Ensure that your configuration gives you the ability to manage files locally by using the command line.
+* Ensure that you have a xref:ccg-fork-the-upstream-repository[copy of the repository on your local system].
 
-.Related information
+* You need the appropriate permissions to create pull requests. Request permission by emailing your GitHub username to ccs-upstream@redhat.com.
```

Comment:
Is this true, though? I think anybody with a fork can create a pull request on GitHub.

Reply:
You know, I don't know.  I thought folks needed permission ... let me see if I can get a new writer to give it a try without permission.

Reply:
I just checked and the repository is set so anyone can 'Read' - so they can create forks, but not push (create pull requests).

Reply:
Sure, they can't push PRs or commits to the base repo, but they can create a PR that others with permissions can push, I believe.

I remember this discussion in the GNOME project about mirroring repos on GitHub that there is no way to disable forking and submitting PRs, which makes the mirroring a bit confusing.  

Seems like this is a feature GitHub won't change, see https://github.com/dear-github/dear-github/issues/84.

Reply:
Thanks for clarifying!